actors/multisig/src/types.rs

```diff
@@ -91,6 +91,9 @@ pub struct ProposeReturn {
     pub ret: RawBytes,
 }
 
+impl Cbor for ProposeParams {}
```

Comment:
@Stebalien can you confirm there are no negative side effects to doing this on param and return types?

Reply:
Yep. It's mostly just a marker.

Reply:
As in, no, no side effects.